packages/adblocker-playwright-example/index.ts

```diff
@@ -17,30 +22,42 @@ import * as pw from 'playwright';
 
   await blocker.enableBlockingInPage(page);
 
-  blocker.on('request-blocked', (request: Request) => {
+  blocker.on('request-blocked', (request) => {
```

Comment:
Why removing the types here?

Reply:
Those types were outdated in the examples. After this PR, I want all event listeners to be typed properly. The types are defined at: https://github.com/ghostery/adblocker/pull/3881/files#diff-0a5765da7fa865c0ab71fc7b0187e3c6663bcc602558022d04dd7da3c0ab410aR97-R109

Reply:
If they were out-dated, can we update them instead of removing? I'd rather keep the type annotations in the examples for documentation purpose.

Reply:
Sure, I like this direction 👍